### PR TITLE
[DOCS] Replace 'time series' with 'timestamped'

### PR DESCRIPTION
**Pull request by kilfoyle**
In the past, we’ve used “time series” to refer to any data with a timestamp. This distinguishes "timestamped" data from "time series" data, which is specific to time series data streams.

### CODE REVIEW

**Comment by kilfoyle** [comment]
@elasticmachine update branch

**Review by abdonpijpelink — APPROVED** [review]
Left a couple of comments about the use of "timestamped" in some contexts. I may be missing background information about switching to this terminology though, so feel free to ignore.

File: docs/reference/data-management.asciidoc [hunk 1]
```diff
@@ -7,13 +7,13 @@
 The data you store in {es} generally falls into one of two categories:
 
 * Content: a collection of items you want to search, such as a catalog of products
-* Time series data: a stream of continuously-generated timestamped data, such as log entries
+* Timestamped data: a stream of continuously-generated timestamped data, such as log entries
```
[the remark this answers is not in the document]
Comment (lockewritesdocs) [reply docs/reference/data-management.asciidoc:10]: I agree that we should use _time series_ here. Here's the definition of [time series data](https://www.elastic.co/guide/en/elastic-stack-glossary/current/terms.html#glossary-time-series-data) from our Glossary:

> Timestamped data such as logs, metrics, and events that is indexed on an ongoing basis.

[the remark this answers is not in the document]
Comment (nik9000) [reply docs/reference/data-management.asciidoc:10]: I love standards! "time series" is the standard name for the kind of thing that we're building, so far as I can tell. But I wouldn't be surprised if it were also the standard name for data how we use it. Maybe it's worth someone doing a review to make sure we're naming things properly.

To be clear, @lockewritesdocs's quote above is what we thought "time series" meant. But the "time series" feature we're building now is more specific to this definition from [wikipedia](https://en.wikipedia.org/wiki/Time_series):
> In [mathematics](https://en.wikipedia.org/wiki/Mathematics), a time series is a series of [data points](https://en.wikipedia.org/wiki/Data_point) indexed (or listed or graphed) in time order. Most commonly, a time series is a [sequence](https://en.wikipedia.org/wiki/Sequence) taken at successive equally spaced points in time. Thus it is a sequence of [discrete-time](https://en.wikipedia.org/wiki/Discrete-time) data. Examples of time series are heights of ocean [tides](https://en.wikipedia.org/wiki/Tides), counts of [sunspots](https://en.wikipedia.org/wiki/Sunspots), and the daily closing value of the [Dow Jones Industrial Average](https://en.wikipedia.org/wiki/Dow_Jones_Industrial_Average).

Specifically, the new "time series" support let's us efficiently consider the series of points together. And it sort of relies on the "successive equally spaced points in time" bit. They don't have to be equally spaced, but the time is part of the identifier for the measure.

**Comment by lockewritesdocs** [comment]
Our glossary defines [time series data](https://www.elastic.co/guide/en/elastic-stack-glossary/current/terms.html#glossary-time-series-data) as:

> Timestamped data such as logs, metrics, and events that is indexed on an ongoing basis. 

Based on that definition, anywhere that discusses **timestamped data that's continually indexed** should used _time series data_. For example, how the [Data tiers](https://www.elastic.co/guide/en/elasticsearch/reference/master/data-tiers.html) docs describe the hot tier:

>[Hot tier](https://www.elastic.co/guide/en/elasticsearch/reference/master/data-tiers.html#hot-tier) nodes handle the indexing load for time series data such as logs or metrics and hold your most recent, most-frequently-accessed data.

To me, that definition refers to timestamped data that's continually indexed, so we should use _time series data_. I think that this logic applies to CCR as well, especially where we're discussing how to configure auto-follow patterns to automatically create follower indices for new time series indices.

All that said, do we want to want to introduce another term here? I feel like these terms are pretty nuanced, and I'm wondering if it's more confusing for users to distinguish between _timestamped_ data and _time series data_.

**Comment by lockewritesdocs** [comment]
@nik9000 and I submitted comments simultaneously 😆 It seems that we're changing our definition of time series means. If that's the case, we should update our definition in the Glossary to reflect this new definition.

**Comment by nik9000** [comment]
> @nik9000 and I submitted comments simultaneously laughing It seems that we're changing our definition of time series means. If that's the case, we should update our definition in the Glossary to reflect this new definition.

:+1:

I do want to double check that it's sensible to change that definition. I sure think it is because I've been thinking about "time series database" stuff for a while. But it'd be good to have someone a little more neutral do some digging.

**Comment by kilfoyle** [comment]
@nik9000, @abdonpijpelink, @lockewritesdocs Thanks for the terrific feedback! I agree with everything that's been said.

So... how about if we keep the term `time series` everywhere and update the glossary instead? Borrowing a bit from the current glossary definition for [data stream](https://www.elastic.co/guide/en/elastic-stack-glossary/current/terms.html#glossary-data-stream), a bit from the mathematics 'time series' definition, and a bit from Nik's nice description of the TSDS, what would you all think of the following?

 **time series data**
A series of data points, such as logs, metrics and events, that is indexed in time order. Time series data can be indexed in a [data stream](https://www.elastic.co/guide/en/elastic-stack-glossary/current/terms.html#glossary-data-stream), where it can be accessed as a single named resource with the data stored across multiple backing indices. A [time series data stream](http://www.link.to.tsds.definition.com/) is optimized for indexing metrics data.

**time series data stream (TSDS)**
A type of [data stream](https://www.elastic.co/guide/en/elastic-stack-glossary/current/terms.html#glossary-data-stream) optimized for indexing metrics [time series data](http://www.google.com/), A TSDS allows for reduced storage size and for a sequence of metrics data points to be considered efficiently as a whole. See [Time series data stream](https://www.elastic.co/guide/en/elasticsearch/reference/master/tsds.html).

**Comment by kilfoyle** [comment]
Closing this PR in favour of [2193](https://github.com/elastic/stack-docs/pull/2193) which updates the Glossary, as proposed and agreed on [above](https://github.com/elastic/elasticsearch/pull/87897#issuecomment-1196107127).

**Pull request closed without merging**